pages/common/distccd.md

```diff
@@ -0,0 +1,24 @@
+# distccd
+
+> Server daemon for distcc distributed compiler.
+> More information: <https://distcc.github.io>.
+
+- Start a daemon with default settings:
+
+`distccd --daemon`
+
+- Start a daemon, accepting connections from private network IPv4 ranges:
+
+`distccd --daemon --allow-private`
+
+- Start a daemon, accepting connections from a specific network address or address range:
+
+`distccd --daemon --allow {{ip_address|network_prefix}}`
+
+- Start a daemon with a maximum of 4 tasks at any time and lowered priority (niceness = 5):
+
+`distccd --daemon --jobs {{4}} --nice {{5}}`
```

Comment:
I think we should either use hard-coded values:
```
`distccd --daemon --jobs 4 --nice 5`
```
or variables:
```
`distccd --daemon --jobs {{maximum_job_number}} --nice {{no_idea_how_to_name_it_but_I_am_sure_you_know}}`
```

Reply:
I experimented with a few formats of this example and found this one to be the cleanest for quick reference.
Also, the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) says:

> If the example is clearer with an actual value rather than a generic placeholder, use the actual value. For example, use iostat {{2}} rather than iostat {{interval_in_secs}}.

Reply:
I think we can keep it like that. One thing we could do is change it to maximum priority. Right now, I don't have a feeling for what the niceness really is. If I knew the maximum value, it would be easier to come up with a lower one. (assuming it goes from 0 to max)

Reply:
I didn't know it was that complicated. In that case I think you got a point with 
> For those who don't know how to handle niceness, there is a ready example how to reduce priority of a daemon below "normal". For those who know what niceness is, that's a clear indication that they can put any valid value they want.

Maybe we should get some other opinions (@bl-ue, @navarroaxel, @nicokosi)